src/node_file.cc

```diff
@@ -394,21 +441,21 @@ class fs_req_wrap {
 void Access(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args.GetIsolate());
   HandleScope scope(env->isolate());
-
-  if (args.Length() < 2)
-    return TYPE_ERROR("path and mode are required");
-  if (!args[1]->IsInt32())
-    return TYPE_ERROR("mode must be an integer");
+  Local<Context> context = env->context();
+  CHECK(args.Length() >= 2 && args[1]->IsInt32());
```

Comment:
Nit: Could we change this to two `Check`s? If one of them fails it's easier to see what fails.

Reply:
@fhinkel Of course!